devicetypes/automateasia/wyfy-touch-zwave-switch.src/wyfy-touch-zwave-switch.groovy

```diff
@@ -0,0 +1,522 @@
+/**
```

Comment:
Since Your devices are going to be integrated in zwave-multi-metering-switch.groovy, this file has to be removed.